spec/support/mvi/find_candidate_failure_response.xml

```diff
@@ -1,20 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  This is a legacy response.
+  It is not representative of an actual response that would contain an 'AR' typeCode.
+  This response (INVALID REQUEST) today (Sept 12, 2019) would contain an 'AE' typeCode.
+  This will be used to test processing 'AR' responses until we can generate a more realistic response.
```

Comment:
What's blocking us from generating a  

> more realistic response

Reply:
'AR' is returned when something is wrong with the MVI service
> AR - fail to process (reject) the message for reasons unrelated to its content or format (system down, internal error, etc.).  For most such problems it is likely that the responding system will be able to accept the same message at a later time.

I can't send a request that causes the MVI backend to simulate a system problem.